src/Datadog.Trace.ClrProfiler.Native/integration_loader.cpp

```diff
@@ -29,7 +29,13 @@ std::vector<Integration> LoadIntegrationsFromFile(const WSTRING& file_path) {
   try {
     std::ifstream stream;
     stream.open(ToString(file_path));
-    integrations = LoadIntegrationsFromStream(stream);
+
+    if (static_cast<bool>(stream)) {
```

Comment:
This change is unrelated to the exclusion list. It replaces useless log message about not being able to deserialize json (if the file is not found) with a message that includes the filename.